foxtrot-core/src/main/java/com/flipkart/foxtrot/core/table/impl/DistributedTableMetadataManager.java

```diff
@@ -278,6 +272,9 @@ private void estimateCardinality(final String table, final Collection<FieldMetad
                     query.addAggregation(AggregationBuilders.percentiles(field)
                             .field(field)
                             .percentiles(10, 20, 30, 40, 50, 60, 70, 80, 90, 100));
+                    /*query.addAggregation(AggregationBuilders.cardinality(field + field)
```

Comment:
@santanusinha @r0goyal Thinking to check cardinality for numbers as well
We can allow queries if the cardinality is less for a numeric field
As of now, we are not considering cardinality of numeric fields and just blocking the query if the grouping is on any numeric field